modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1120,6 +1122,36 @@ CV__DNN_INLINE_NS_BEGIN
      */
     CV_EXPORTS_W Net readNetFromONNX(const std::vector<uchar>& buffer);
 
+     /** @brief This struct contains all the configuration of TensorRT backend.
+      */
+    struct CV_EXPORTS_W_SIMPLE TrtConfig
+    {
+        CV_PROP_RW int deviceId;      //!< Running TensorRT with specific GPU
+        CV_PROP_RW bool useCache;     //!< It will speed up the ONNX model initialization. Write back the converted trt file to where the input model is located.
+        CV_PROP_RW String cachePath;   //!< Setting the path to write back the converted .trt file to specific path instead of the input file location.
```

Comment:
Can cache be controlled by environment variable or using default OpenCV cache?

Reply:
Yes, we can use enviroment variable to control it, the question is TensorRT has a lot of options, we can't set all of optionals by enviroment variable like [ONNXRuntime done](https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#environment-variablesdeprecated) previously. Some TensorRT used by ONNXRuntime: https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_info.cc#L15-L58